docs/_documentations/troubleshooting.md

```diff
@@ -1010,3 +1011,20 @@ Install and enable the [Java Extension Pack](https://marketplace.visualstudio.co
 
 ### Debugger fails to attach after restarting project into Debug mode
 Run the `attach debugger` command again. If the issue persists after a few attempts, restart the project in **Debug** mode a second time.
+
+***
+# Restoring lost connections
```

Comment:
Is there a section for debug somewhere that this can be put under? I am just concerned about the possible confusion between a `connection` to a remote codewind instance and a debug `connection`.